rest/health_test.go

```diff
@@ -74,10 +92,24 @@ func TestNewHealth(t *testing.T) {
 					"success": {
 						Status:  HealthStatusHealthy,
 						Message: "",
+						Services: map[string]HealthResult{
```

Comment:
Can we add one test where we have a nested service structure and the status of the parent is derived automatically?

Reply:
It's already tested.